services/src/main/java/org/keycloak/broker/oidc/OIDCIdentityProvider.java

```diff
@@ -389,6 +389,37 @@ public BrokeredIdentityContext getFederatedIdentity(String response) {
                 throw new IdentityBrokerException("Mismatch between the subject in the id_token and the subject from the user_info endpoint");
             }
 
+            if (getConfig().isFilteredByClaims()) {
+                String filterName = getConfig().getClaimFilterName();
+                String filterValue = getConfig().getClaimFilterValue();
+
+                logger.infof("Filtering user %s by %s=%s", idToken.getOtherClaims().get(getusernameClaimNameForIdToken()), filterName, filterValue);
```

Comment:
Is it possible to use `tracef` instead of `infof` (in this place and also in all other in this class) and also use `debugf` instead of `warnf` (in this place and also in all other in this class). As AFAIK `info` and `warn` are usually for special messages or server-wide messages/issues, but not for regular "user" requests.

Reply:
Thanks for sharing these practices